vars.tf

```diff
@@ -75,4 +75,8 @@ output minions {
 
 output minion_security_group {
     value = "${aws_security_group.minions.id}"
-}
\ No newline at end of file
+}
+
+output availability_zones_in_use {
+    value = "${aws_instance.minion.*.availability_zone}"
```

Comment:
Hmm I think that'll output too many; what you want is `"${aws_subnet.main.*.availability_zone}"`